site/operational-guide.markdown

```diff
@@ -0,0 +1,384 @@
+# Concepts
+
+This section describes some essential concepts with which you will
+need to be familiar before moving on to the example deployment
+scenarios.
+
+> Author's Note: I have deliberately excluded any mention of
+> weave command syntax in this section
+
+## Peer
+
+A peer is a host on which you have installed weave.
+
+## Peer Name
+
+Peers in the weave network are identified by a 48-bit value formatted
+like an ethernet MAC address e.g. `01:23:45:67:89:ab`. This 'peer
+name' is used for various purposes:
+
+* Routing of packets between containers on the overlay network
+* Recording the origin peer of DNS entries
+* Recording ownership of IP address ranges
+
+Whilst the first two uses tolerate the peer name changing on router
```

Comment:
Good point - I have removed the references to lost address space here. I also added in information on `/sys/hypervisor/uuid` which we also fall back to now.